util/internal/test/common/src/main/java/org/hibernate/search/util/impl/test/ThrowingFunction.java

```diff
@@ -0,0 +1,12 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.util.impl.test;
```

Comment:
I'm sorry, I made a mistake in my comment on JIRA; I meant these function classes should live in package `org.hibernate.search.util.impl.test.function`. Could you please move them there in an additional commit? Using your IDE's refactoring tools, it should be easy :)

Reply:
😏 ok